packages/pcre/pcre.7.1.6/files/pcre.install

```diff
@@ -0,0 +1,6 @@
+etc: [
+  "setup.ml"
+  "setup.data"
+  "setup.log"
+  "_oasis_remove_.ml"
+]
```

Comment:
This is to use oasis scripts to remove packages so that it works correctly under Windows.  For this, the oasis files need to be preserved.  The version of oasis2opam in the dev repo (unreleased) uses "ocamlfind" when only libraries are installed.  I can make a new release if you want.

On Fri, 21 Aug 2015 08:00:41 -0700, Markus Mottl wrote:
> 
> @Chris00 It seems that these lines were added by the latest oasis2opam. I'm not sure all of these need to be installed. Could you please let me know what to do about it?